### PR TITLE
Fix cointracking import

### PR DESCRIPTION
**Pull request by markusw**
Fix for #731

### CODE REVIEW

**Comment by LefterisJP** [comment]
Hey @markusw thank you so much for the PR! Can you also add a commit with a changelog entry explaining the bug that's fixed here: https://github.com/rotki/rotki/blob/ad3d80c032618c5c88b5909f52254f57b1288c14/docs/changelog.rst#changelog ?

For the format just check the other bugs.

**Comment by markusw** [comment]
Hey, hope thats right, I'm pretty new at this.
Also updated the link on the contribute page because it doesn't work anymore. :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Rotkehclhen Contributing Guide
 
-The rotkhelhchen contribution guide is hosted in readthedocs. Click [here](https://rotkehlchen.readthedocs.io/en/latest/contribute.html) to go there.
+The rotkhelhchen contribution guide is hosted in readthedocs. Click [here](https://rotki.readthedocs.io/en/latest/contribute.html) to go there.
```
Comment (LefterisJP) [inline CONTRIBUTING.md]: Aaah nice catch!

**Comment by LefterisJP** [comment]
> Hey, hope thats right, I'm pretty new at this.

It's perfect! Thank you so much!

> Also updated the link on the contribute page because it doesn't work anymore. :)

Very nice catch!

---

I am merging this with my thanks! Hope to see you stick around if you would like to contribute or use Rotki in the future!

**Pull request merged**